src/constants/formTest.constants.js

```diff
@@ -0,0 +1,6 @@
+export const SIGNUP_FIELDS = {
+  FIRST_NAME: 'John',
+  LAST_NAME: 'Dee',
+  EMAIL: 'john.dee@someemail.com',
+  PASSWORD: 'Alex_12345'
+};
```

Comment:
Not sure we need this constant at all

Reply:
@VladWithCookies why not? on Sign In form tests we will need email & password

Reply:
@VladWithCookies i just renamed it to - AUTH_FIELDS

Reply:
It is just a mock data for the tests, it is not very useful to store it in the constant